### ISSUE TITLE
[Request]: Option to force Vulkan ICD loader (This fixed Ubisoft Connect for me!)

### ISSUE THREAD

**Issue by swebow** [bottlesdevs/Bottles#3178]
### Tell us the problem or your need

I have had an massive issue with Ubisoft Connect for I don't know how long now.
Today I just got fed up and started to play with it in both Lutris and Bottles.

After awhile I managed to stumble upon a thread that told me how to force one GPU to be used instead of the other for Vulkan loading. I did those settings suggested and ran with the wine-ge-8-24-x86_64 runner. Boom! Ubisoft Connect works with my Discrete Graphic card. Glorius 100+ fps in Far Cry Blood Dragon (Fast game to download and start, why it was my test subject)

Since my computer use a integrated AMD and a Discrete Nvidia, it's of course more complicated. But if that option was possible in Bottles as well, it should help a lot of people!

![2023-11-28_20-16](https://github.com/bottlesdevs/Bottles/assets/48765498/56e31695-cfed-404e-a769-0df2ef1e3ce7)

### Describe the solution you'd like

If possible, copy the option from Lutris to force enable Vulkan ICD loader option! Put it maybe under "advanced". Ubisoft Connect is the only application that have been a headache for me.

**Comment by xioren** [comment]
I think this would be a useful addition and would simultaneously solve issues such as #3133. I am currently looking into what it would take to implement.

**Comment by averagenative** [comment]
Would also help with #3078 . I have this problem primarily with apps that are launchers.

**Comment by SirDK** [comment]
Good idea. Had same problem because of nvidia 545 crap driver. Much of headache.

**Comment by xioren** [comment]
Question: Is it enough to be able to choose the GPU Bottles uses or do you guys actually need/want the ability to choose the Vulkan ICD files?

**Comment by swebow** [comment]
I think you already can pick the ICD right now? Just that even if I for example pick Vulkan in there. It refuse to run on my Discrete graphics card.So I guess the ability to pick Graphic card? :)

![image](https://github.com/bottlesdevs/Bottles/assets/48765498/c73e2e58-4e93-4385-93cb-df63633abcb4)

**Comment by swebow** [comment]
> Question: Is it enough to be able to choose the GPU Bottles uses or do you guys actually need/want the ability to choose the Vulkan ICD loader?

I think you already can pick the ICD right now? Just that even if I for example pick Vulkan in there. It refuse to run on my Discrete graphics card.So I guess the ability to pick Graphic card? :)

![image](https://github.com/bottlesdevs/Bottles/assets/48765498/959c0b28-6533-46ac-a14b-c7e7cd790110)
